### DISCUSSION TITLE
I have been approved but I do not have the benefits package yet

### DISCUSSION THREAD

**Discussion by isMMa** [community/community#132557] [GitHub Education]
### Hi everyone,

I have been approved but I do not have the benefits package yet:

Hi, isMMa! You were last verified as a student on Jun 21, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

**Comment by Moosems** [comment]
Its currently July 17th, and it generally takes a week or so for this to be set up.

**Comment by isMMa** [comment]
Okay, although we've been there for almost a month

**Reply by Moosems** [reply]
My bad, I misread the n in Jun as an l as in Jul and was confused because it it. Yeah, I’d probably just call tech support.
